u2f-chrome-extension/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "FIDO U2F (Universal 2nd Factor) extension",
   "description": "Provides the FIDO U2F APIs for authentication. PRE-RELEASE",
-  "version": "0.9.22",
+  "version": "0.9.30",
```

Comment:
Super nit: upstream has changed to 0.9.31, could you remerge? It'll make finding the appropriate branch point a little easier.